docs/bin/build-preview-site.sh

```diff
@@ -51,11 +51,17 @@ cp -r $SOURCE_DIR/doc/index.md $JEKYLL_DIR/doc
 
 echo "Setting up LB4 doc pages",
 rm -rf $JEKYLL_DIR/pages
-ln -s $PWD/site $JEKYLL_DIR/pages
+# Create hardlinks because Jekyll does not support symbolic links any more.
```

Comment:
There's a need to add 
` "realpath": "^1.1.0",`
to devDependencies

Reply:
Weird. I believe `realpath` is a Unix tool, this is what I see on my machine:

```
$ ls -l `which realpath`
lrwxr-xr-x  1 bajtos  admin  37 Sep 26 15:26 /usr/local/bin/realpath -> ../Cellar/coreutils/8.31/bin/realpath
```

See https://www.gnu.org/software/coreutils/manual/html_node/realpath-invocation.html

Anyhow, I guess it's better to explicitly install realpath as a Node.js dependency, rather than rely on `coreutils` being installed.

Reply:
FYI - I was able to run this script without installing `realpath`.